### DISCUSSION TITLE
如何指定默认ppgw？

### DISCUSSION THREAD

**Discussion by rewei** [kkkgo/PaoPaoGateWay#64] [问和答]
现在我使用两个不同的ppgw来做国外流量的分流，请问如何指定其中一个为默认的ppgw呢？

**Comment by kkkgo** [comment]
也许你可以转换下思路，把两个线路合并到一个yaml里面去。

**Comment by rewei** [comment]
哈哈，感谢大佬的回答，我之前就是用一个ppgw，再用clash的规则来对国外流量分流的，今天看到您在另一个解答中谈到用两个ppgw的提议就试了一下，结果还是可以的，但是如果不能指定默认ppgw，后边就会乱跑了。https://github.com/kkkgo/PaoPaoGateWay/discussions/37#discussioncomment-7605744

**Reply by kkkgo** [reply]
我那个的意思是不同的fakeip的cidr段走不同的ppgw。比如6.0.0.0/8走10.10.10.4，7.0.0.0/8走10.10.10.5，这样是不会乱跑或者冲突的。然后你用静态路由或者域名分流到其中一个cidr段就可以了。

**Comment by rewei** [comment]
还是有点转不过弯，我试验做的似乎和你说的一样，算了，还是回归最初吧，谢谢您的解答
